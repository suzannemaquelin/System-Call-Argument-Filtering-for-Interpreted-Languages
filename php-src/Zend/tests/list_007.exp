Fatal error: Uncaught Error: Cannot use object of type Closure as array in %slist_007.php:3
Stack trace:
#0 {main}
  thrown in %slist_007.php on line 3