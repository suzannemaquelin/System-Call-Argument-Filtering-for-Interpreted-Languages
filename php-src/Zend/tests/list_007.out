Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Fatal error: Uncaught Error: Cannot use object of type Closure as array in /root/php-src/Zend/tests/list_007.php:3
Stack trace:
#0 {main}
  thrown in /root/php-src/Zend/tests/list_007.php on line 3